app/src/main/java/org/greenstand/android/TreeTracker/activities/MainActivity.kt

```diff
@@ -232,4 +251,15 @@ class MainActivity : AppCompatActivity(), ActivityCompat.OnRequestPermissionsRes
         locationUpdateManager.startLocationUpdates()
         locationDataCapturer.start()
     }
+
+    override fun onAccuracyChanged(sensor: Sensor?, accuracy: Int) {
+        // Ignore
```

Comment:
Wish there was a default implementation for this making it unnecessary to declare it here. Not sure if we  really care about change in accuracy reported for step count. Ignoring the implementation.